### ISSUE TITLE
Jumbled dataframes

### ISSUE THREAD

**Issue by greyarmy** [FTHuld/neware_reader#11]
Hello FTHuld,

I have recently started processing Neware cycling data for the first time, I was pleased to find your package!!

Unfortunately, about 5-10% of the time the script outputs a garbage data frame with all the data jumbled up. I have double-checked the data files using the Neware software so it is not the samples that have failed.

Have you come across this problem before?

**Comment by FTHuld** [comment]
Hi @greyarmy.

I suspect it's an issue with the version of neware you're using. If the location of the data in the bytestream is shifted slightly between the version I originally wrote this for and the one you're using then the output will be off, which then outputs gibberish.

I'm away from my desk this week, but if you send me a small file that is showing the output your describing I can take a look in the next few weeks.

Alternatively you could open the file in something like HxD and see if you can figure out how the data is structured. The bytestream should still be 86 bytes wide.

**Comment by greyarmy** [comment]
Hi @FTHuld,

I have had more time with the exported data and have now realised all of it is being jumbled up.

Here is an example file of the data I have been trying to export, I took a look at the file with HxD and got as far as figuring out that the byte stream is still 86 bytes wide.

Thank you for taking the time to have a look at it.


[example_file.zip](https://github.com/FTHuld/neware_reader/files/9838871/example_file.zip)

**Comment by FTHuld** [comment]
Hi @greyarmy.

I tried to run your file on your machine, and I'm not seeing any obvious problems?
I tried running 
```
import neware_reader.neware as nda
nda.nda_to_csv('ex_file.nda')
```
Which gave me the following output:
[ex_file.nda.csv](https://github.com/FTHuld/neware_reader/files/9857728/ex_file.nda.csv)

Are you seeing something different?

I have noticed that opening the cell data on the BTS software sometimes gives nonsense data. Could that be the issue? In these cases I normally re-open the data from the BTS and it's usually fixed.
